src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -462,26 +464,28 @@ const PreviewHeader = ({
 }
 
 // This component is responsible for rendering the html preview of the analysis file
-const AnalysisPreviewFrame = ({ analysisName, toolLabel, workspace: { workspace: { workspaceId, googleProject, bucketName } }, onRequesterPaysError, styles }) => {
+const AnalysisPreviewFrame = ({ analysisName, toolLabel, workspace, onRequesterPaysError, styles }) => {
   const signal = useCancellation()
   const [busy, setBusy] = useState(false)
   const [preview, setPreview] = useState()
   const frame = useRef()
+  const cloudPlatform = getCloudProviderFromWorkspace(workspace)
+  const { workspace: { workspaceId, googleProject, bucketName } } = workspace
```

Comment:
thanks. A pit pedantic, but I find using workspace likes this makes it easier when it comes time to type the props!